### PR TITLE
Document how to configure multiple engine installations to run on a same computer

### PR DESCRIPTION
**Pull request by ptormene**
Fixes https://github.com/gem/oq-engine/issues/8725

### CODE REVIEW

File: doc/adv-manual/general.rst [hunk 1]
```diff
@@ -1896,3 +1896,127 @@ the engine version. Here are a few examples::
...
+    calc_id     calculation ID [default: -1]
+
+  options:
+    -h, --help  show this help message and exit
+
+If the key `/` is requested, the root attributes are retrieved,
+i.e. `checksum`, `date` `engine_version` and `input_size`.
+If the calculation id is not specified, the value of the
+requested key is retrieved for the latest calculation.
+
```
Comment (micheles) [inline doc/adv-manual/general.rst:1923]: I would add an example here

File: doc/adv-manual/general.rst [hunk 2]
```diff
@@ -1896,3 +1896,127 @@ the engine version. Here are a few examples::
...
+  covered by other models
+* `OQ_EXCLUDE_MODELS`: same as above, but selecting sites covered by all models
+  except those specified in this list
+* `OQ_ONLY_SITEIDS`: a comma-separated list of site identifiers to be selected,
+  excluding all others from the analysis
+* `OQ_EXCLUDE_SITEIDS`: a comma-separated list of site identifiers to be excluded,
+  selecting all the others
+* `OQ_MAX_SITES_PER_MODEL`: the maximum quantity of sites to be selected between
+  those covered by each model
+
```
Comment (micheles) [inline doc/adv-manual/general.rst:1976]: I don't like depending on environment variables (IMNSHO they are meant for debugging, NOT for regular usage), however this is another discussion, independent from this PR.

File: doc/faq.md [hunk 3]
```diff
@@ -74,12 +74,59 @@ You can also limit the number of parallel threads as explained
...
+HDF5 dataset is independent from all others in the datastore and it has a unique
+identifier. It is possible to determine the version of the engine that produced
+each HDF5 file (`calc_<calc_id>.hdf5`) using the command
+
+```
+$ oq show_attrs / <calc_id>
+```
+
+where `/` indicates the root attributes (`checksum`, `date`
+`engine_version` and `input_size`) and `<calc_id>` (an integer number) is the
```
Comment (micheles) [inline doc/faq.md:126]: I would write (`date`, `engine_version`, etc) so we do not need to update the docs if new attributes will be added in the future.

File: doc/adv-manual/general.rst (hunk 2 above)
Comment (ptormene) [reply doc/adv-manual/general.rst]: Done

File: doc/faq.md (hunk 3 above)
Comment (ptormene) [reply doc/faq.md]: Done

File: doc/adv-manual/general.rst (hunk 1 above)
Comment (ptormene) [reply doc/adv-manual/general.rst:1931]: Done

**Review by micheles — APPROVED** [review]

**Pull request merged**